netatmo.js

```diff
@@ -135,6 +135,15 @@ netatmo.prototype.authenticate = function (args, callback) {
   return this;
 };
 
+/**
+ * @returns access_token
+ */
+netatmo.prototype.getAccessToken = function () {
+  
+  return access_token
```

Comment:
Should check to see if there is an `access_token` first and then return it. Otherwise should return `false` or an error.

Thoughts @ilblog?